dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -15,7 +15,8 @@ def index
 
     ROLES.each do |role|
       apps = get_applications_by_role(role).
-        select(:status, :regional_partner_id, "COUNT(locked_at) as total_locked", "COUNT(id) as total")
+        select(:status, "IF(locked_at IS NULL, FALSE, TRUE) AS total_locked", "count(id) as total").
```

Comment:
nit: why `AS total_locked`? Shouldn't that just be `AS locked` and then below `.group(:status, :locked)`? It's calculated per row, and used in the grouping, rather than aggregated across all rows in the bucket like `count(id) AS total` so total doesn't seem appropriate.

Also capitalize the final `as`

Reply:
I was using the pre-existing variable names, but I see your point and I'll make that change.